### PR TITLE
Add SIGMA 16mm F1.4 DC DN | C for Sony E from upload e34e56

### PR DESCRIPTION
**Pull request by tgurr**
Fixes <#544>.

### CODE REVIEW

**Comment by tgurr** [comment]
Still would like to hear if there's some better approach than to have the entry two times for darktable and RawTherapee to properly auto-detect the right lensfun entry. Everything else I tried, like two model entries in one dataset failed :cry:.

**Comment by bronger** [comment]
Two entries should be merged because Lensfun is supposed to be neutral with respect to the application program.  You may try the union of both names, i.e. something that contains all tokens.

Sigma 16mm f/1.4 DC DN | Contemporary C 017

**Comment by tgurr** [comment]
@bronger thanks! That appears to work indeed for both of them, updated the PR.

File: data/db/mil-sigma.xml [hunk 1]
```diff
@@ -224,4 +224,15 @@
         </calibration>
     </lens>
 
+    <lens>
+        <maker>Sigma</maker>
+        <model>Sigma 16mm f/1.4 DC DN | Contemporary C 017</model>
+        <mount>Sony E</mount>
```
Comment (bronger) [inline data/db/mil-sigma.xml]: Is this really the only mount?

Comment (tgurr) [reply data/db/mil-sigma.xml]: The lense is also available for Micro 4/3, but that one is already covered by https://github.com/lensfun/lensfun/commit/dc7557d5c8f5bc62e572ef1d462efdfcf87a5107 and has different calibration data as far as I can judge, also the results doesn't look good for my photos when choosing the already avalable Micro 4/3 entry manually. I'm new to this stuff so I might be missing how to properly combine data within the xml.

Comment (bronger) [reply data/db/mil-sigma.xml]: Ah, good to know. We do the following in such cases: Put the two datasets next to each other, add all mounts to both entries. In a future Lensfun version, we hope to be able to merge such duplicates.

Comment (tgurr) [reply data/db/mil-sigma.xml]: > Put the two datasets next to each other

check
> add all mounts to both entries

did that for the new entry, not sure if adding emount to the already existing should be part of this PR. I could do a follow-up PR or add the emount to the existing entry with a second commit in this PR. What do you prefer?

Comment (bronger) [reply data/db/mil-sigma.xml]: You’re right, I do that separately.

**Pull request merged**